internal/cli/apis.go

```diff
@@ -249,7 +280,16 @@ auth0 apis update <id|audience> --name myapi`,
 				return err
 			}
 
-			api := &management.ResourceServer{}
+			if err := apiTokenLifetime.AskU(cmd, &inputs.TokenLifetime, nil); err != nil {
+				return err
+			}
+
+			apiAllowOfflineAccess.AskBoolU(cmd, &inputs.AllowOfflineAccess, nil)
+
+			api := &management.ResourceServer{
```

Comment:
This is an update command, please use the current existing values as the default unless new ones have been specified. Please take a look at the other params below for reference.